src/mame/arcade.flt

```diff
@@ -1222,6 +1222,7 @@ spaceg.cpp
 spartanxtec.cpp
 spbactn.cpp
 spcforce.cpp
+spdamges.cpp
```

Comment:
This doesn't match the name of the driver (spdamjes.cpp). And same in mame.lst.

Reply:
I used the wrong branch, it's an unrelated work, I've reverted the commit.